### PR TITLE
[mimecast] use queried fields

### PR DESCRIPTION
**Pull request by dorschw**
related: [XSUP-25208](https://jira-hq.paloaltonetworks.local/browse/XSUP-25208)

### CODE REVIEW

**Comment by dorschw** [comment]
See the closing reason for [XSUP-25208](https://jira-hq.paloaltonetworks.local/browse/XSUP-25208).

File: Packs/Mimecast/ReleaseNotes/2_1_11.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+
+#### Integrations
+
+##### Mimecast v2
+
+- Fix an issue where **mimecast-query** command does not return additional keys that not appear in the default xml query.
```
Comment (dorschw) [inline Packs/Mimecast/ReleaseNotes/2_1_11.md:6]:
```suggestion
When calling **mimecast-query**, fields specified under `<return-fields>` now appear in context and readable output, alongside the default fields that were shown until now.
```

File: Packs/Mimecast/pack_metadata.json [hunk 2]
```diff
@@ -2,7 +2,7 @@
     "name": "Mimecast",
     "description": "Mimecast unified email management offers cloud email services for email security, continuity and archiving emails. Read the detailed instructions to understand how to configure the integration's parameters.",
     "support": "xsoar",
-    "currentVersion": "2.1.10",
+    "currentVersion": "2.1.11",
```
Comment (dorschw) [inline Packs/Mimecast/pack_metadata.json:5]:
```suggestion
    "currentVersion": "2.2.0",
```
Please change in the release notes as well

**Review by dorschw — COMMENTED** [review]
@shmuel44

File: Packs/Mimecast/pack_metadata.json (hunk 2 above)
Comment (shmuel44) [reply Packs/Mimecast/pack_metadata.json:5]: Done

File: Packs/Mimecast/ReleaseNotes/2_1_11.md (hunk 1 above)
Comment (shmuel44) [reply Packs/Mimecast/ReleaseNotes/2_1_11.md:6]: Done

File: Packs/Mimecast/Integrations/MimecastV2/MimecastV2_test.py [hunk 3]
```diff
@@ -566,3 +593,32 @@ def test_list_email_queues_command(mocker):
...
+        - Calling query()
+    Then
+        - Make sure all return-field values are returned to context and human-readable.
+    """
+    
+    query_data = util_load_json("test_data/query_response.json")
+    mocker.patch.object(MimecastV2, "http_request", return_value=query_data["response"])
+    mocker.patch.object(demisto, "args", return_value={"queryXml": QUERY_XML})
+    result = MimecastV2.query()
+    result.get("outputs")
```
[the remark this answers is not in the document]
Comment (shmuel44) [reply Packs/Mimecast/Integrations/MimecastV2/MimecastV2_test.py:619]: i del this, Done

File: Packs/Mimecast/ReleaseNotes/2_2_0.md [hunk 4]
```diff
@@ -0,0 +1,6 @@
+
+#### Integrations
+
+##### Mimecast v2
+
+- When calling **mimecast-query**, fields specified under `<return-fields>` now appear in context and readable output, alongside the default fields that were shown until now.
```
Comment (dorschw) [inline Packs/Mimecast/ReleaseNotes/2_2_0.md:6]:
```suggestion
When calling **mimecast-query**, fields specified under `<return-fields>` now appear in context and readable output, alongside the default fields that were shown until now.
```

**Review by shmuel44 — APPROVED** [review]

File: Packs/Mimecast/ReleaseNotes/2_2_0.md [hunk 5]
```diff
@@ -0,0 +1,6 @@
+
+#### Integrations
+
+##### Mimecast v2
+
+When calling **mimecast-query**, fields specified under `<return-fields>` now appear in context and readable output, alongside the default fields that were shown until now.
```
Comment (dorschw) [inline Packs/Mimecast/ReleaseNotes/2_2_0.md:6]:
```suggestion
Added support for displaying specified fields in context and readable output when calling **mimecast-query**. These fields are specified under <return-fields> and will now appear alongside the default fields that were shown until now.
```

**Pull request merged**